Funcky/Functional/ActionToUnit.cs

```diff
@@ -7,74 +7,83 @@ public static partial class Functional
     {
         [Pure]
         public static Func<Unit> ActionToUnit(Action action)
-            => () =>
-            {
-                action();
-                return Unit.Value;
-            };
+            => ()
+                =>
```

Comment:
Is this reformatting intentional?

Reply:
Yes, the arrow operator is at the beginning of a line not the end.